x-pack/plugins/reporting/export_types/common/lib/screenshots.js

```diff
@@ -59,9 +59,16 @@ export function screenshotsObservableFactory(server) {
     await browser.waitForSelector(`${layout.selectors.renderComplete},[${layout.selectors.itemsCountAttribute}]`);
   };
 
```

Comment:
I had it as-is because we can't reliably differentiate "error" toast messages from any other kind of toast. So if "error" is not going to refer to what we're looking for on the page. This wouldn't be do-able if we don't also grab the message from the toast. The user can review what was there and decide if configuration needs to be fixed, or if was just a flash message and they can re-try again.

How about `checkForToastMessage`?

Reply:
Sounds good to me, and agreed